src/Standards/Squiz/Sniffs/Arrays/ArrayDeclarationSniff.php

```diff
@@ -620,20 +619,32 @@ public function processMultiLineArray($phpcsFile, $stackPtr, $arrayStart, $array
                     continue;
                 }
 
-                if ($lastValueLine !== false && $tokens[$value['value']]['line'] === $lastValueLine) {
+                $valuePointer = $value['value'];
+
+                if ($tokens[$valuePointer]['line'] === $tokens[$stackPtr]['line']) {
+                    $error = 'The first value in a multi-value array must be on a new line';
```

Comment:
Does this mean the code 
```
$a = array($b, $c, $d,
$e);
```
Will return three times the same error ?
One time for the value $b, one time for $c and one time for $d.

Reply:
I didn't add this code, I only moved it to the more appropriate position. And no, this code deals only with the first element of the array. And in general, afaik, yes - each misplaced element gets its own error. However, I'm not sure as I really didn't focus on that.

Reply:
![image](https://user-images.githubusercontent.com/9052536/72204490-c986dc80-3478-11ea-96b7-5136f003b648.png)

I try my example, and got the error multiple times.

The check $previous === false should be added.

Reply:
Good catch with that example. I took inspiration in your next comment but fixed it a little differently. Anyway, it's now simpler and works better :)